packages/lime-app/Makefile

```diff
@@ -48,3 +48,53 @@ define Package/$(PKG_NAME)/postinst
 endef
 
 $(eval $(call BuildPackage,$(PKG_NAME)))
+#
+# Copyright (C) Libremesh 2017
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lime-app
+PKG_VERSION:= 3ea6d26c7c789b0224d0cf9a9f500b7a93b34c9985277d61f798e35022412741
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=v0.2.0-alpha.5
+PKG_SOURCE_URL:=https://github.com/libremesh/lime-app/releases/download/$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+	CATEGORY:=LiMe
+	TITLE:=LimeApp
+	MAINTAINER:=Marcos Gutierrez <gmarcos@altermundi.net>
+	URL:=http://github.com/libremesh/lime-app
+	DEPENDS:=+uhttpd +uhttpd-mod-ubus \
+		+ubus-lime-location +ubus-lime-metrics +ubus-lime-utils \
+		+ubus-lime-openairview +ubus-lime-grondrouting
+	PKGARCH:=all
+endef
+
+define Package/$(PKG_NAME)/description
+	Light webApp for LibreMesh over uhttpd
+endef
+
+define Build/Compile
+endef
+
+define Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/www/app/
+	$(CP) $(BUILD_DIR)/build/* $(1)/www/app/
+	$(INSTALL_DIR) $(1)/etc/uci-defaults
+	$(INSTALL_BIN) ./files/lime-app.defaults $(1)/etc/uci-defaults/90_lime-app
+	$(INSTALL_BIN) ./files/95-lime-app-rpc-acl $(1)/etc/uci-defaults/
+	@mkdir -p $(1)/usr/share/rpcd/acl.d || true
```

Comment:
Shouldn't this be $(INSTALL_DIR) ?

Reply:
INSTALL_BIN set chmod +x after copy?

Reply:
@gmarcos87 yes, that's the difference between INSTALL_BIN and INSTALL_DATA

Reply:
Ok, those files are executable scripts so I think it's okay INSTALL_BIN

Reply:
> Shouldn't this be $(INSTALL_DIR) ?

I refer to this line
`@mkdir -p $(1)/usr/share/rpcd/acl.d || true`

Reply:
INSTALL_DIR will create the whole path if there is no intermediate directory?

Reply:
INSTALL_DIR is defined in rules.mk as  `INSTALL_DIR:=install -d -m0755`. It creates all the intermediate directories (like mkdir -p)